modules/upgrade/nav-upgrade-guide.adoc

```diff
@@ -19,6 +19,7 @@ endif::[]
 ** xref:proxy-update.adoc[Proxy Update]
 ** xref:proxy-migration.adoc[Proxy Upgrade 3.x to 4.x]
 ** xref:client-migration.adoc[Client Migration]
-** xref:db-migration.adoc[Database Migration]
+** xref:db-migration.adoc[Database Migration to PostgreSQL 12]
+** xref:db-migration.adoc[Database Migration (Old)]
```

Comment:
Do we need to remove the old one? If we're keeping it, can we rename it?

Reply:
ok, renamed it